DelvUI/Interface/GeneralElements/CastbarHud.cs

```diff
@@ -27,8 +28,8 @@ public class CastbarHud : ParentAnchoredDraggableHudElement, IHudElementWithActo
 
         public CastbarHud(CastbarConfig config, string? displayName = null) : base(config, displayName)
         {
-            _castNameLabel = new LabelHud(config.CastNameConfig);
-            _castTimeLabel = new LabelHud(config.CastTimeConfig);
+            _castNameLabel = new LabelHud(config.CastNameLabel);
+            _castTimeLabel = new LabelHud(config.CastTimeLabel);
         }
 
         public void StopPreview()
```

Comment:
Line #45 doesn't need to be unsafe I believe, there's no pointers in DrawChildren. That's not new since this update though I just noticed it now when going through the file, thus the comment at another line :P
Same thing with the method on line #112 

Also a similar thing goes on line #100 where the variable is never used

So yeah, neither of that is a review of actually new code sorry haha